src/praisonai/praisonai/bots/_durable_adapter.py

```
@@ -0,0 +1,161 @@
+                    outbox=self.outbox,
+                    adapter=self,  # Adapter must implement MessageSender protocol
+                    platform=platform,
+                    max_attempts=max_attempts,
+                )
+                
+                logger.info(
+                    f"[{platform}] Durable delivery enabled: {outbox_path} "
+                    f"(pending={self.outbox.pending_count()})"
+                )
+                
+            except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)